python/pyspark/sql/functions.py

```diff
@@ -69,6 +69,7 @@ def _(col):
     globals()[_name] = _create_function(_name, _doc)
 del _name, _doc
 __all__ += _functions.keys()
```

Comment:
can we add a line of comment to __all__ to say that _functions are automatically added to the list.